edx/edx-ora2
# UI Changes for Rubric Reuse

**TL;DR -** Add a dropdown section to the rubric tab for rubric copying

JIRA: [AU-22](https://openedx.atlassian.net/browse/AU-22)

**What changed?**

- Added a dropdown to the rubric tab
- Added a "combobox"-style autocomplete to the dropdown, with all "other" ORA block ids
- Added a button that when clicked clones the specified rubric into the edit modal
- Added a message that pops up to confirm the copied rubric

**Screenshots**
![base](https://user-images.githubusercontent.com/1639231/126808266-ad31b549-0aa9-4c5a-9567-a8e816b4832d.png)
![dropdown](https://user-images.githubusercontent.com/1639231/126808277-ad0598a2-cd49-4dc9-815c-e824dfd53045.png)
![alert](https://user-images.githubusercontent.com/1639231/126808288-0a8d6c1b-9cd7-49de-8555-ebb77504cb43.png)


**Developer Checklist**


**Testing Instructions**

- Create a new course.
- Import this course: 
[course.ok28jmzh.tar.gz](https://github.com/edx/edx-ora2/files/6840710/course.ok28jmzh.tar.gz)
- In waffle flag course overrides, create a force on override for `openresponseassessment.rubric_reuse` for your course ID.
- Go to the course in studio, and navigate to the COMMAND CENTER -> USE ME TO COPY RUBRICS ora
- Click edit -> rubric
- The dropdown should be there, expanded, and you should be able to choose from a dropdown of block ids. 
- Select various blocks, and copy their  rubrics.

**Reviewer Checklist**

Collectively, these should be completed by reviewers of this PR:


FYI: @edx/masters-devs-gta
